02-FlexBox/assignment/gulpfile.js

```diff
@@ -5,35 +5,37 @@ const {
     series,
     watch,
     task
-} = require('gulp')
+} = require('gulp');
```

Comment:
including these changes is just fine for now, but changing this could cause problems that make other things more challenging. Make sure you know what you're doing to make this work, if you're hacking this to make your code work, it's a sign that you're not doing things quite right.

I'm not too worried, but just want to make sure that changing files like this isn't a habit.